docs/chapters/workflow/release-process.rst

```diff
@@ -0,0 +1,63 @@
+Release process
+===============
+
+Component attributes
+--------------------
+To properly map a release of the PELUX Baseline, we collect a number of
+attributes for the components that make up a release. This goes further than
+only the artifacts produced in the release, since an artifact can be an
+aggregate of several components (such as yocto layers).
+
+Versioning
+    The PELUX Baseline uses a release versioning scheme similar to what Debian
+    uses for their releases. A version number is X.Y where X and Y are integers.
+    Most releases will be X.0, with bug fixes and security updates in subsequent
+    X.Y releases. Any more substantial development done after a release will go
+    into the next release.
+
```

Comment:
Do we want to mention semantic versioning here? https://semver.org/

Reply:
I originally wrote about semver, but since a platform can't fulfill many of the properties required in semantic versioning, I looked at how larger systems (such as Debian) does it. Still, this is more or less a semantic versioning, but there is no API and no concept of "breaking changes" etc.